tuf/conf.py

```diff
@@ -54,6 +54,10 @@
 # be saved to 'LOG_FILENAME'
 LOG_FILENAME = 'tuf.log'
 
+# This is the format/encoding of metadata that TUF will use. The options are
+# 'json' and 'der'. DER is an encoding of ASN.1 used for the Uptane project.
+METADATA_FORMAT = 'json' # 'der'
```

Comment:
Yes; currently, the formats of all metadata files must be consistent.
They're all JSON or all DER. (Except the map (still pinned.json in this
fork) file, which is never signed and sent anywhere. That's just always
JSON in this codebase.)